src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java

```diff
@@ -63,6 +69,9 @@ public void send(PaymentCommand cmd) {
 
     private void doSend(IMessage message, boolean retry) throws ServiceBusException, InterruptedException {
         try {
+            LOG.info("Sending message to payments queue. ID: {}, Label: {}",
```

Comment:
How about pitting this one after `queueClient.send(message);`? We've got a long entry for before it happens.

Reply:
done